### ISSUE TITLE
CAN we customize any website we need 

### ISSUE THREAD

**Issue by pretech86** [thewhiteh4t/seeker#70]
thanks for the tool , but can we use any web site we need instead of this web site if yes how we can do this ? 
thanks a lot

**Comment by thewhiteh4t** [comment]
yes you can use any website you want, just keep the javascript and php intact, don't change any variables, if you really want to change variables then carefully analyse both javascript and php and you will be able to use your own variables, this is a great idea i will give it a try, it will be something like SET

**Comment by pretech86** [comment]
So i will scraped the website i need then what after that , where can i put the website contents 
Sorry I’m just a beginner

**Comment by thewhiteh4t** [comment]
wait for sometime i will add an option to use your own template

**Comment by pretech86** [comment]
Thanks for kindly interest

**Comment by MANOELPIMENTA** [comment]
Friend thewhiteh4t, first of all I would like to congratulate you on the tool provided! I need your guidance!
I would like to know if there is a possibility that we can send the target a link other than the default link to disguise the attack.
If so, guide me how to proceed.
Already read the file README.md, did the tests, but could not succeed, because always gives error.
I await your return, please!

**Comment by royrogers** [comment]
Websites templates are under /seeker/template

/opt/seeker/template# ls
nearyou  sample.kml

copy nearyou as twitter (new template name as you want)

/opt/seeker/template# cp -r nearyou/ /opt/seeker/template/twitter
/opt/seeker/template# ls
nearyou  sample.kml  twitter

navigate inside the twitter folder

/opt/seeker/template# cd twitter/
/opt/seeker/template/twitter# ls
css  index.html  js  php

and edit the index.html as you want
/opt/seeker/template/twitter# geany index.html

finally edit the seeker.py script

#result = 'template/nearyou/php/result.txt'
#info = 'template/nearyou/php/info.txt'
#site = 'nearyou'

result = 'template/twitter/php/result.txt'
info = 'template/twitter/php/info.txt'
site = 'twitter'

#php_rqst = requests.get('http://127.0.0.1:8080/nearyou/index.html')
php_rqst = requests.get('http://127.0.0.1:8080/twitter/index.html'

python3 seeker.py -t default -s twitter

or

python3 seeker.py -t default

**Comment by cool3477** [comment]
> Websites templates are under /seeker/template
> 
> /opt/seeker/template# ls
> nearyou sample.kml
> 
> copy nearyou as twitter (new template name as you want)
> 
> /opt/seeker/template# cp -r nearyou/ /opt/seeker/template/twitter
> /opt/seeker/template# ls
> nearyou sample.kml twitter
> 
> navigate inside the twitter folder
> 
> /opt/seeker/template# cd twitter/
> /opt/seeker/template/twitter# ls
> css index.html js php
> 
> and edit the index.html as you want
> /opt/seeker/template/twitter# geany index.html
> 
> finally edit the seeker.py script
> 
> #result = 'template/nearyou/php/result.txt'
> #info = 'template/nearyou/php/info.txt'
> #site = 'nearyou'
> 
> result = 'template/twitter/php/result.txt'
> info = 'template/twitter/php/info.txt'
> site = 'twitter'
> 
> #php_rqst = requests.get('http://127.0.0.1:8080/nearyou/index.html')
> php_rqst = requests.get('http://127.0.0.1:8080/twitter/index.html'
> 
> python3 seeker.py -t default -s twitter
> 
> or
> 
> python3 seeker.py -t default



dear i dont wana use serveo,  i wana upload template files on hosting server and i need to get victim result in a text file which uploaded on hosting server whenever victim open my web link.. as u know domain hosting servers always keep alive . 
r these changes possible? and i dont wana change any current functionality of seeker for using on domain hosting.

**Comment by Ushan12** [comment]
How to whatsapp Contact dump bro

**Comment by devilimut** [comment]
I have a problem that if the target user rejects the location, google map cannot render the map.
is there a way without a browser confirmation? and most users choose to deny that access

**Comment by ZoOoOoOoOoOom** [comment]
> > Websites templates are under /seeker/template
> > /opt/seeker/template# ls
> > nearyou sample.kml
> > copy nearyou as twitter (new template name as you want)
> > /opt/seeker/template# cp -r nearyou/ /opt/seeker/template/twitter
> > /opt/seeker/template# ls
> > nearyou sample.kml twitter
> > navigate inside the twitter folder
> > /opt/seeker/template# cd twitter/
> > /opt/seeker/template/twitter# ls
> > css index.html js php
> > and edit the index.html as you want
> > /opt/seeker/template/twitter# geany index.html
> > finally edit the seeker.py script
> > #result = 'template/nearyou/php/result.txt'
> > #info = 'template/nearyou/php/info.txt'
> > #site = 'nearyou'
> > result = 'template/twitter/php/result.txt'
> > info = 'template/twitter/php/info.txt'
> > site = 'twitter'
> > #php_rqst = requests.get('http://127.0.0.1:8080/nearyou/index.html')
> > php_rqst = requests.get('http://127.0.0.1:8080/twitter/index.html'
> > python3 seeker.py -t default -s twitter
> > or
> > python3 seeker.py -t default
> 
> dear i dont wana use serveo, i wana upload template files on hosting server and i need to get victim result in a text file which uploaded on hosting server whenever victim open my web link.. as u know domain hosting servers always keep alive . r these changes possible? and i dont wana change any current functionality of seeker for using on domain hosting.

Did you reach to something?
I'm looking for the same i think the solution is to setup kali linux on vps but did you have a nother idea like ngrok record the results?

**Comment by Resousse** [comment]
Hello @ZoOoOoOoOoOom , sorry there is no video, but I let you compare with one easy existing template (zoom or telegram template)

**Comment by MAGNUM999** [comment]
Hello all, I am new to this and I am wondering is it possible to customise the domain name instead of using ngrok domain so is less suspicious? My other goal is to clone a site.

**Comment by thewhiteh4t** [comment]
@MAGNUM999 yes you can setup nginx proxy and use a custom domain with it

**Comment by eemotionn** [comment]
Hello all ,  I am new to this and I want to know if the results from Seeker can be mapped to a web page, allowing direct access via a map to retrieve and display the location.

**Comment by thewhiteh4t** [comment]
@eemotionn if you are talking about something like a dashboard then yes it's a work in progress at the moment, you can share the features you are thinking about and I can try adding those before release
